rts/Rml/Backends/RmlUi_Renderer_GL3.cpp

```diff
@@ -1,5 +1,7 @@
+/* This file is part of the Spring engine (GPL v2 or later), see LICENSE.html */
+
 /*
- * This source file is part of RmlUi, the HTML/CSS Interface Middleware
+ * This source file is dervied from the source code of RmlUi, the HTML/CSS Interface Middleware
```

Comment:
t. dervish
```suggestion
 * This source file is derived from the source code of RmlUi, the HTML/CSS Interface Middleware
```

but also idk if it makes sense to modify this. did we actually change anything substantial?